.github/workflows/reviewdog.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Run Misspell with Reviewdog
         uses: reviewdog/action-misspell@v1
         with:
-          github_token: ${{ secrets.GH_ACCESS_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
           reporter: github-pr-review
```

Comment:
This will probably have to become a `github-pr-check`

Reply:
It works also with `github-pr-review` and I think comments are a bit more readable.

Reply:
Oh, in that case we might want to change back the vale linter to review comments

Reply:
Good call, thank you! I forgot to add it.